src/api/app/views/webui/feeds/notifications.rss.builder

```diff
@@ -0,0 +1,26 @@
+xml.rss version: '2.0' do
+  xml.channel do
+    xml.title "#{@user.realname} (#{@user.login}) notifications"
+    xml.description "Event notifications from #{@configuration['title']}"
+    xml.link url_for only_path: false, controller: 'main', action: 'index'
+    xml.language "en"
+    xml.pubDate Time.now
+    xml.generator @configuration['title']
+
+    @notifications.each do |notification|
+      xml.item do
+        xml.title notification.event.subject
+        # We render the event_mailer view to 'description'
+        xml.description ApplicationController.renderer.new.render(
```

Comment:
Anybody who comes along in the future and sees `ApplicationController.renderer` getting instantiated in a view will wonder why its like that and will probably investigate it which would be a waste of developer time.

You can just add this line here to [`Webui::FeedsController#notifications`](https://github.com/mdeniz/open-build-service/blob/84cf7620cd82d1643a8cce78b8db7a7d44d4e0ab/src/api/app/controllers/webui/feeds_controller.rb#L47) and then get rid of the assigns and use a normal `render` call like its done usually:

```ruby
@host = ::Configuration.obs_url
```